projects/angular/src/wizard/wizard-stepnav-item.ts

```diff
@@ -158,4 +170,30 @@ export class ClrWizardStepnavItem {
       throw new Error('Wizard stepnav item is not associated with a wizard page.');
     }
   }
+
+  private ensureCurrentStepIsScrolledIntoView() {
+    const stepnavItemElement = this.elementRef.nativeElement;
+    const stepnavWrapperElement = stepnavItemElement.closest<HTMLDivElement>('.clr-wizard-stepnav-wrapper');
+
+    return this.navService.currentPageChanged.pipe(
+      tap(currentPage => {
+        if (
+          currentPage === this.page &&
+          !elementIsScrolledIntoView({ container: stepnavWrapperElement, element: stepnavItemElement })
+        ) {
+          stepnavItemElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
+        }
+      })
+    );
+  }
+}
+
+function elementIsScrolledIntoView({ container, element }: { container: HTMLElement; element: HTMLElement }) {
```

Comment:
Is there specific need for paasing a object instead of params?
The naming of the method can be improved.
Check the suggestion.

```suggestion
function elementIsInView(container: HTMLElement, element: HTMLElement) {
```

Reply:
>Is there specific need for paasing a object instead of params?

Yes. It prevents transposing the parameters.